packages/react-native-editor/src/api-fetch-setup.js

```diff
@@ -7,9 +7,11 @@ import { applyFilters } from '@wordpress/hooks';
 
 const SUPPORTED_METHODS = [ 'GET', 'POST' ];
 // Please add only wp.org API paths here!
+// Don't add /themes endpoint due to an issue on Android
+// with the wp.org API implementation for React Native.
```

Comment:
It may be worth publishing a comment on https://github.com/wordpress-mobile/WordPress-Android/issues/21005 capturing the identified issue and linking to it within this code comment. Alternative, inlining an explanation in the code comment would work well too. WDYT?

Reply:
Good idea! I'll update it 👍

Reply:
Updated in https://github.com/WordPress/gutenberg/pull/63183/commits/7a5d33c555f1eef1add380a40d4f75eef885b1dd